src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -157,7 +163,7 @@ class HuiEntityButtonCard extends hassLocalizeLitMixin(LitElement)
     return `hsl(${hue}, 100%, ${100 - sat / 2}%)`;
   }
 
-  private handleClick() {
+  private handleClick(hold=false) {
```

Comment:
why add a default value?

Reply:
... I... don't know... It was needed in some case before, but I've since then changed things so it isn't...